src/Altinn.Platform/Altinn.Platform.Events/Tests/TestingUtils/AuthorizationHelperTests.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Xacml.JsonProfile;
+using Altinn.Platform.Events.Authorization;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.UnitTest.Mocks;
+using AltinnCore.Authentication.Constants;
+using Xunit;
+
+namespace Altinn.Platform.Events.Tests.TestingUtils
+{
+    /// <summary>
+    /// /
+    /// </summary>
+    public class AuthorizationHelperTests
+    {
+        /// <summary>
+        /// Test access to own event
+        /// </summary>
+        [Fact]
+        public async Task AuthorizeAccessToEventForSelf()
+        {
+            PepWithPDPAuthorizationMockSI pdp = new PepWithPDPAuthorizationMockSI();
+            AuthorizationHelper authzHelper = new AuthorizationHelper(pdp);
+
+            CloudEvent cloudEvent = new CloudEvent()
+            {
+                Source = new Uri("https://skd.apps.altinn.no/ttd/endring-av-navn-v2/instances/1337/6fb3f738-6800-4f29-9f3e-1c66862656cd"),
+                Subject = "/party/1337"
+            };
+
+            // Act
+
+            bool result = await authzHelper.AuthorizeConsumerForAltinnAppEvent(cloudEvent, "/user/1337");
+
+            // Assert.
+            Assert.True(result);
+        }
+
+        /// <summary>
+        /// Test access to own event
+        /// </summary>
+        [Fact]
+        public async Task AuthorizeOrgAccessToEventForUser()
+        {
+            PepWithPDPAuthorizationMockSI pdp = new PepWithPDPAuthorizationMockSI();
+            AuthorizationHelper authzHelper = new AuthorizationHelper(pdp);
+
+            CloudEvent cloudEvent = new CloudEvent()
+            {
+                Source = new Uri("https://skd.apps.altinn.no/ttd/endring-av-navn-v2/instances/1337/6fb3f738-6800-4f29-9f3e-1c66862656cd"),
+                Subject = "/party/1337"
+            };
+
+            // Act
+
```

Comment:
Doesn't extra empty lines like this produce a warning? Try to fix as many warnings as possible.